js/containers/App.jsx

```diff
@@ -120,11 +108,22 @@ function App(props) {
 
 const mapStateToProps = function(state) {
 	return {
-		message: state.message,
-		messageType: state.message_type,
-		activeStep: state.activeStep,
+		isLoading: [
+			state.git.is_loading || state.git.is_updating,
+			state.plan.is_loading,
+			false,
```

Comment:
tbh, this whole section is a bit of a place holder until we know how it will connect with the 'real' menu